Vagrantfile

```diff
@@ -17,9 +17,6 @@ Vagrant.configure('2') do |config|
 
   config.vm.provision :chef_solo do |chef|
     chef.json = {
-      :riemann => {
-        :version => '0.2.4'
```

Comment:
No, it's not used anywhere I'm aware of. I switched to using the package URL, since that's the only place the version number is used.